### PR TITLE
Skosgi util

### PR DESCRIPTION
**Pull request by skosgi1**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: src/classes/UTIL_Describe.cls [hunk 1]
```diff
@@ -560,11 +613,15 @@ public class UTIL_Describe {
...
     public static String getHhAccRecTypeID() {
         String recTypeId = getRecTypesMapByDevName('Account').get('HH_Account');
+
         if(String.isBlank(recTypeId) && Test.isRunningTest()) {
+
             if(getRecTypesMapByDevName('Account').values().size() > 1) {
                 return getRecTypesMapByDevName('Account').values()[1];
             }
-            else return null;
+            else {
```
Comment (lifewithryan) [inline src/classes/UTIL_Describe.cls]: @skosgi1 would you mind bringing this "else" up to the line above it so we have "cuddled" elses?  e.g. } else {

**Comment by robertbwatson** [comment]
@skosgi1 @lifewithryan I noticed this PR is pointing from skosgiUtil to feature/skosgi/indentationDescribeUtil. Ideally, the commits would have been made on the feature/skosgi/indentationDescribeUtil branch and the PR would be pointing from feature/skosgi/indentationDescribeUtil to master.

I think this might be okay for right now - once this PR is merged, then the commits will all be in the feature/skosgi/indentationDescribeUtil branch. Then we can create another pull request to merge feature/skosgi/indentationDescribeUtil into master.

**Comment by lifewithryan** [comment]
> @skosgi1 @lifewithryan I noticed this PR is pointing from skosgiUtil to feature/skosgi/indentationDescribeUtil. Ideally, the commits would have been made on the feature/skosgi/indentationDescribeUtil branch and the PR would be pointing from feature/skosgi/indentationDescribeUtil to master.
> 
> I think this might be okay for right now - once this PR is merged, then the commits will all be in the feature/skosgi/indentationDescribeUtil branch. Then we can create another pull request to merge feature/skosgi/indentationDescribeUtil into master.

Yep. That's exactly what we have to do :) Its alot like when we get a PR from the community. We pull it down and put it into a EDA branch and merge to master from there. In the future we should just create the branch directly from EDA. I am a command line person so from master: git checkout -b feature/<username>/featureName, do work, commit, push with --set-upstream if necessary (usually is on the first one) then submit PR.

**Comment by skosgi1** [comment]
@lifewithryan I have updated the requested changes, can you please give it another shot and let me know? @robertbwatson @lifewithryan Thank you for letting me know. I will follow the process you mentioned from my next WI onwards.

File: src/classes/UTIL_Describe.cls [hunk 2]
```diff
@@ -618,8 +618,7 @@ public class UTIL_Describe {
 
             if(getRecTypesMapByDevName('Account').values().size() > 1) {
                 return getRecTypesMapByDevName('Account').values()[1];
-            }
-            else {
+            }else {
```
Comment (lifewithryan) [inline src/classes/UTIL_Describe.cls]: @skosgi1 SUPER tiny fix. put a space before the else here. } else {

Comment (skosgi1) [reply src/classes/UTIL_Describe.cls]: @lifewithryan My bad. fixed it.

**Comment by robertbwatson** [comment]
@lifewithryan Do you think we need any release notes for this?

**Comment by lifewithryan** [comment]
Going to merge this into the feature branch. Then I will create a PR from that and perhaps have someone review from there.

**Pull request merged**